tests/autoyast/prepare_cloned_profile.pm

```diff
@@ -21,7 +21,7 @@ use autoyast qw(inject_registration expand_variables upload_profile);
 
 sub run {
     # Get path in the worker
-    my $path = get_var('ASSETDIR') . '/other/' . get_var('ASSET_1');
+    my $path = get_required_var(get_required_var('AUTOYAST'));
```

Comment:
This entire PR should be reduced to a one-liner:

    -my $path = get_var('ASSETDIR') . '/other/' . get_var('ASSET_1');
    +my $path = get_var('ASSET_1');

The changes I suggested for yast2_clone_system are unnecessary, the `ASSET_1` filename will be expanded correctly. Yiannis got the error in his verification run because the asset is marked as private and is accessible only inside the job group. Create verification runs with `openqa-clone-job` and set `CASEDIR` manually.

Reply:
Alright, then I will leave only this change as `get_required_var('AUTOYAST')` contains `ASSET_1`. Thanks!